etc/config.xml

```diff
@@ -38,6 +37,16 @@
                 <enable_oneclick>1</enable_oneclick>
                 <enable_recurring>0</enable_recurring>
                 <group>adyen</group>
+                <!-- etc/adminhtml/system/order_management/order_management.xml -->
+                <order_status>pending</order_status>
+                <payment_pre_authorized>pending</payment_pre_authorized>
+                <payment_authorized>processing</payment_authorized>
+                <payment_authorized_virtual>complete</payment_authorized_virtual>
+                <payment_cancelled>canceled</payment_cancelled>
+                <fraud_manual_review_status>pending</fraud_manual_review_status>
+                <fraud_manual_review_accept_status>processing</fraud_manual_review_accept_status>
```

Comment:
@msilvagarcia I see that in the prototype this field is set to Pending and not to Processing, but Pending is not an option on the source model we're using for it (Magento\Sales\Model\Config\Source\Order\Status\Processing). Seems like a mistake on the prototype. Maybe it is a good idea to state this in the ticket so the discrepancy is documented?

Reply:
Done!